<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataViewPage</class>
 <widget class="QWidget" name="dataViewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLineEdit" name="searchBar_lineEdit">
       <property name="minimumSize">
        <size>
         <width>381</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>381</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="inventoryData_pushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 173, 181);
font: 700 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Inventory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="staffData_pushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 173, 181);
font: 700 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Staff</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="patientData_pushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 173, 181);
font: 700 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Patient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="medicineData_pushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 173, 181);
font: 700 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Medicine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prescriptionData_pushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 173, 181);
font: 700 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Prescription</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logData_pushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 173, 181);
font: 700 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Logs</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="dataView_tableView">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
